### ISSUE TITLE
generate:entity-boilerplate errors output in auto_install.sql

### ISSUE THREAD

**Issue by rbaugh** [totten/civix#249]
So this is somewhat in reply to ticket #237. I didn't get a chance to test the patch before it was merged but have tested and started to get some odd errors. I am getting the following error code dumped into my auto_install.sql file.

```
Warning: Trying to access array offset on value of type null in /private/var/folders/bn/zjh6cq791rb9n1gb0p62m1v00000gn/T/templates_c_QEmK7y.d/%%29^298^298C3DBD%%schema.tpl.php on line 17

Call Stack:
16788509087.6185     356056   1. {main}() /usr/local/bin/civix:0
16788509087.6185     594656   2. require('phar:///usr/local/bin/civix/bin/civix') /usr/local/bin/civix:14
16788509087.6185     673272   3. CRM\CivixBundle\Application::main() phar:///usr/local/bin/civix/bin/civix:28
16788509087.6185    1056000   4. CRM\CivixBundle\Application->run() phar:///usr/local/bin/civix/src/CRM/CivixBundle/Application.php:38
16788509087.6185    1149840   5. CRM\CivixBundle\Application->doRun() phar:///usr/local/bin/civix/vendor/symfony/console/Application.php:149
16788509087.6185    1153224   6. CRM\CivixBundle\Application->doRunCommand() phar:///usr/local/bin/civix/vendor/symfony/console/Application.php:273
16788509087.6185    1153224   7. CRM\CivixBundle\Command\AddEntityBoilerplateCommand->run() phar:///usr/local/bin/civix/vendor/symfony/console/Application.php:1009
16788509087.6185    1155056   8. CRM\CivixBundle\Command\AddEntityBoilerplateCommand->execute() phar:///usr/local/bin/civix/vendor/symfony/console/Command/Command.php:255
16788509087.6185   43879200   9. CRM\CivixBundle\Command\AddEntityBoilerplateCommand->CRM\CivixBundle\Command\{closure:phar:///usr/local/bin/civix/src/CRM/CivixBundle/Command/AddEntityBoilerplateCommand.php:118-137}() phar:///usr/local/bin/civix/src/CRM/CivixBundle/Command/AddEntityBoilerplateCommand.php:138
16788509087.6185   43879392  10. CRM_Core_CodeGen_Schema->generateCreateSql() phar:///usr/local/bin/civix/src/CRM/CivixBundle/Command/AddEntityBoilerplateCommand.php:125
16788509087.6185   43886896  11. CRM_Core_CodeGen_Util_Template->fetch() /Volumes/rbaugh/civi-training/app/public/wp-content/plugins/civicrm/civicrm/CRM/Core/CodeGen/Schema.php:61
16788509087.6185   43886896  12. Smarty->fetch() /Volumes/rbaugh/civi-training/app/public/wp-content/plugins/civicrm/civicrm/CRM/Core/CodeGen/Util/Template.php:106
16788509087.6185   43960080  13. include('/private/var/folders/bn/zjh6cq791rb9n1gb0p62m1v00000gn/T/templates_c_QEmK7y.d/%%29^298^298C3DBD%%schema.tpl.php') /Volumes/rbaugh/civi-training/app/public/wp-content/plugins/civicrm/civicrm/packages/Smarty/Smarty.class.php:1273


Variables in local scope (#13):
  $_block_content = *uninitialized*
  $_block_repeat = *uninitialized*
  $_from = *uninitialized*
  $_smarty_tpl_vars = *uninitialized*
  $_tmp = *uninitialized*
```

I have tried to clear the templates before hand, but some of this is coming from Civi core files. This is running on 5.51.0 release of Civi Core.

**Comment by rbaugh** [comment]
I should note that I had this happen right when the fix for issue #237 was added in and got the error. I have re-pulled civix as well before posting the issue just to make sure and I still get the same code added to the auto_install.sql.

This is from v22.07.0 of civix

**Comment by totten** [comment]
I played around for a while to figure out how to reproduce. Basic steps were:

* Check out an older version of civix (eg `v21.04.1`)
* Generate an example (`generate:module`, `generate:upgrader`, `generate:entity` and finally `generate:entity-boilerplate`)
* Switch to newer civix (eg `v22.07.0`)
* Re-run `generate:entity-boilerplate`

These steps don't always produce the erroroneous SQL -- but if you use php80 and WordPress, then it does. (It seems php74 isn't as less likely to raise warnings, and WP registers a more verbose error-handler.) For me, the error in `sql/auto_install.mysql` looks nearly identical, and (additionally) there are some CLI warnings:

```
[bknix-edge:~/bknix/build/wpmaster/web/wp-content/plugins/civicrm/civicrm/ext/myent] ~/src/civix/bin/civix generate:entity-boilerplate

Warning: Undefined array key "tableAttributes_simple" in /Users/totten/bknix/build/wpmaster/web/wp-content/plugins/civicrm/civicrm/CRM/Core/CodeGen/Specification.php on line 226

Call Stack:
    0.0002     413784   1. {main}() /Users/totten/src/civix/bin/civix:0
    0.0041     825224   2. CRM\CivixBundle\Application::main($binDir = '/Users/totten/src/civix/bin') /Users/totten/src/civix/bin/civix:29
    0.0238    2147448   3. CRM\CivixBundle\Application->run($input = ???, $output = ???) /Users/totten/src/civix/src/CRM/CivixBundle/Application.php:38
    0.0313    2461072   4. CRM\CivixBundle\Application->doRun($input = class Symfony\Component\Console\Input\ArgvInput { private $tokens = [0 => 'generate:entity-boilerplate']; private $parsed = []; protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $hasAnArrayArgument = FALSE; private $hasOptional = FALSE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = []; protected $arguments = ['command' => 'generate:entity-boilerplate']; protected $interactive = TRUE }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private $stderr = class Symfony\Component\Console\Output\StreamOutput { private $stream = resource(96) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... } }; private $consoleSectionOutputs = []; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(89) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } } }) /Users/totten/src/civix/vendor/symfony/console/Application.php:149
    0.0313    2464456   5. CRM\CivixBundle\Application->doRunCommand($command = class CRM\CivixBundle\Command\AddEntityBoilerplateCommand { private ${Symfony\Component\Console\Command\Command}application = class CRM\CivixBundle\Application { private ${Symfony\Component\Console\Application}commands = [...]; private ${Symfony\Component\Console\Application}wantHelps = FALSE; private ${Symfony\Component\Console\Application}runningCommand = ...; private ${Symfony\Component\Console\Application}name = 'civix'; private ${Symfony\Component\Console\Application}version = '@package_version@'; private ${Symfony\Component\Console\Application}commandLoader = NULL; private ${Symfony\Component\Console\Application}catchExceptions = TRUE; private ${Symfony\Component\Console\Application}autoExit = TRUE; private ${Symfony\Component\Console\Application}definition = class Symfony\Component\Console\Input\InputDefinition { ... }; private ${Symfony\Component\Console\Application}helperSet = class Symfony\Component\Console\Helper\HelperSet { ... }; private ${Symfony\Component\Console\Application}dispatcher = NULL; private ${Symfony\Component\Console\Application}terminal = class Symfony\Component\Console\Terminal { ... }; private ${Symfony\Component\Console\Application}defaultCommand = 'list'; private ${Symfony\Component\Console\Application}singleCommand = FALSE; private ${Symfony\Component\Console\Application}initialized = TRUE }; private ${Symfony\Component\Console\Command\Command}name = 'generate:entity-boilerplate'; private ${Symfony\Component\Console\Command\Command}processTitle = NULL; private ${Symfony\Component\Console\Command\Command}aliases = []; private ${Symfony\Component\Console\Command\Command}definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $hasAnArrayArgument = FALSE; private $hasOptional = FALSE; private $options = [...]; private $shortcuts = [...] }; private ${Symfony\Component\Console\Command\Command}hidden = FALSE; private ${Symfony\Component\Console\Command\Command}help = 'Creates DAOs, mysql install and uninstall instructions, and an appropriate\nhook_civicrm_entityTypes based on this extension\'s xml schema files.\n\nTypically you will run this command after creating or updating one or more\nxml/schema/CRM/NameSpace/EntityName.xml files.\n'; private ${Symfony\Component\Console\Command\Command}description = 'Generates boilerplate code for entities based on xml schema definition files (*EXPERIMENTAL AND INCOMPLETE*)'; private ${Symfony\Component\Console\Command\Command}ignoreValidationErrors = FALSE; private ${Symfony\Component\Console\Command\Command}applicationDefinitionMerged = TRUE; private ${Symfony\Component\Console\Command\Command}applicationDefinitionMergedWithArgs = TRUE; private ${Symfony\Component\Console\Command\Command}code = NULL; private ${Symfony\Component\Console\Command\Command}synopsis = ['short' => 'generate:entity-boilerplate [options]', 'long' => 'generate:entity-boilerplate [--yes]']; private ${Symfony\Component\Console\Command\Command}usages = []; private ${Symfony\Component\Console\Command\Command}helperSet = class Symfony\Component\Console\Helper\HelperSet { private $helpers = [...]; private $command = NULL } }, $input = class Symfony\Component\Console\Input\ArgvInput { private $tokens = [0 => 'generate:entity-boilerplate']; private $parsed = []; protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $hasAnArrayArgument = FALSE; private $hasOptional = FALSE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = []; protected $arguments = ['command' => 'generate:entity-boilerplate']; protected $interactive = TRUE }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private $stderr = class Symfony\Component\Console\Output\StreamOutput { private $stream = resource(96) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... } }; private $consoleSectionOutputs = []; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(89) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } } }) /Users/totten/src/civix/vendor/symfony/console/Application.php:273
    0.0314    2464456   6. CRM\CivixBundle\Command\AddEntityBoilerplateCommand->run($input = class Symfony\Component\Console\Input\ArgvInput { private $tokens = [0 => 'generate:entity-boilerplate']; private $parsed = []; protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $hasAnArrayArgument = FALSE; private $hasOptional = FALSE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = []; protected $arguments = ['command' => 'generate:entity-boilerplate']; protected $interactive = TRUE }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private $stderr = class Symfony\Component\Console\Output\StreamOutput { private $stream = resource(96) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... } }; private $consoleSectionOutputs = []; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(89) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } } }) /Users/totten/src/civix/vendor/symfony/console/Application.php:1009
    0.0314    2466288   7. CRM\CivixBundle\Command\AddEntityBoilerplateCommand->execute($input = class Symfony\Component\Console\Input\ArgvInput { private $tokens = [0 => 'generate:entity-boilerplate']; private $parsed = []; protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $hasAnArrayArgument = FALSE; private $hasOptional = FALSE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = []; protected $arguments = ['command' => 'generate:entity-boilerplate']; protected $interactive = TRUE }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private $stderr = class Symfony\Component\Console\Output\StreamOutput { private $stream = resource(96) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... } }; private $consoleSectionOutputs = []; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(89) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } } }) /Users/totten/src/civix/vendor/symfony/console/Command/Command.php:255
    0.2433   39855088   8. CRM_Core_CodeGen_Specification->getTable($tableXML = class SimpleXMLElement { public $base = 'CRM/Myent'; public $class = 'Foo'; public $name = 'civicrm_foo'; public $comment = 'FIXME'; public $log = 'true'; public $field = [0 => class SimpleXMLElement { ... }, 1 => class SimpleXMLElement { ... }]; public $primaryKey = class SimpleXMLElement { public $name = 'id'; public $autoincrement = 'true' }; public $foreignKey = class SimpleXMLElement { public $name = 'contact_id'; public $table = 'civicrm_contact'; public $key = 'id'; public $onDelete = 'CASCADE' } }, $database = ['tableAttributes_modern' => 'ENGINE=InnoDB'], $tables = NULL) /Users/totten/src/civix/src/CRM/CivixBundle/Command/AddEntityBoilerplateCommand.php:86

Write /Users/totten/bknix/build/wpmaster/web/wp-content/plugins/civicrm/civicrm/ext/myent/CRM/Myent/DAO/Foo.php
Write /Users/totten/bknix/build/wpmaster/web/wp-content/plugins/civicrm/civicrm/ext/myent/sql/auto_install.sql

Warning: Undefined property: stdClass::$database in /Users/totten/bknix/build/wpmaster/web/wp-content/plugins/civicrm/civicrm/CRM/Core/CodeGen/Schema.php on line 55

Call Stack:
    0.0002     413784   1. {main}() /Users/totten/src/civix/bin/civix:0
    0.0041     825224   2. CRM\CivixBundle\Application::main($binDir = '/Users/totten/src/civix/bin') /Users/totten/src/civix/bin/civix:29
    0.0238    2147448   3. CRM\CivixBundle\Application->run($input = ???, $output = ???) /Users/totten/src/civix/src/CRM/CivixBundle/Application.php:38
    0.0313    2461072   4. CRM\CivixBundle\Application->doRun($input = class Symfony\Component\Console\Input\ArgvInput { private $tokens = [0 => 'generate:entity-boilerplate']; private $parsed = []; protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $hasAnArrayArgument = FALSE; private $hasOptional = FALSE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = []; protected $arguments = ['command' => 'generate:entity-boilerplate']; protected $interactive = TRUE }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private $stderr = class Symfony\Component\Console\Output\StreamOutput { private $stream = resource(96) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... } }; private $consoleSectionOutputs = []; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(89) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } } }) /Users/totten/src/civix/vendor/symfony/console/Application.php:149
    0.0313    2464456   5. CRM\CivixBundle\Application->doRunCommand($command = class CRM\CivixBundle\Command\AddEntityBoilerplateCommand { private ${Symfony\Component\Console\Command\Command}application = class CRM\CivixBundle\Application { private ${Symfony\Component\Console\Application}commands = [...]; private ${Symfony\Component\Console\Application}wantHelps = FALSE; private ${Symfony\Component\Console\Application}runningCommand = ...; private ${Symfony\Component\Console\Application}name = 'civix'; private ${Symfony\Component\Console\Application}version = '@package_version@'; private ${Symfony\Component\Console\Application}commandLoader = NULL; private ${Symfony\Component\Console\Application}catchExceptions = TRUE; private ${Symfony\Component\Console\Application}autoExit = TRUE; private ${Symfony\Component\Console\Application}definition = class Symfony\Component\Console\Input\InputDefinition { ... }; private ${Symfony\Component\Console\Application}helperSet = class Symfony\Component\Console\Helper\HelperSet { ... }; private ${Symfony\Component\Console\Application}dispatcher = NULL; private ${Symfony\Component\Console\Application}terminal = class Symfony\Component\Console\Terminal { ... }; private ${Symfony\Component\Console\Application}defaultCommand = 'list'; private ${Symfony\Component\Console\Application}singleCommand = FALSE; private ${Symfony\Component\Console\Application}initialized = TRUE }; private ${Symfony\Component\Console\Command\Command}name = 'generate:entity-boilerplate'; private ${Symfony\Component\Console\Command\Command}processTitle = NULL; private ${Symfony\Component\Console\Command\Command}aliases = []; private ${Symfony\Component\Console\Command\Command}definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $hasAnArrayArgument = FALSE; private $hasOptional = FALSE; private $options = [...]; private $shortcuts = [...] }; private ${Symfony\Component\Console\Command\Command}hidden = FALSE; private ${Symfony\Component\Console\Command\Command}help = 'Creates DAOs, mysql install and uninstall instructions, and an appropriate\nhook_civicrm_entityTypes based on this extension\'s xml schema files.\n\nTypically you will run this command after creating or updating one or more\nxml/schema/CRM/NameSpace/EntityName.xml files.\n'; private ${Symfony\Component\Console\Command\Command}description = 'Generates boilerplate code for entities based on xml schema definition files (*EXPERIMENTAL AND INCOMPLETE*)'; private ${Symfony\Component\Console\Command\Command}ignoreValidationErrors = FALSE; private ${Symfony\Component\Console\Command\Command}applicationDefinitionMerged = TRUE; private ${Symfony\Component\Console\Command\Command}applicationDefinitionMergedWithArgs = TRUE; private ${Symfony\Component\Console\Command\Command}code = NULL; private ${Symfony\Component\Console\Command\Command}synopsis = ['short' => 'generate:entity-boilerplate [options]', 'long' => 'generate:entity-boilerplate [--yes]']; private ${Symfony\Component\Console\Command\Command}usages = []; private ${Symfony\Component\Console\Command\Command}helperSet = class Symfony\Component\Console\Helper\HelperSet { private $helpers = [...]; private $command = NULL } }, $input = class Symfony\Component\Console\Input\ArgvInput { private $tokens = [0 => 'generate:entity-boilerplate']; private $parsed = []; protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $hasAnArrayArgument = FALSE; private $hasOptional = FALSE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = []; protected $arguments = ['command' => 'generate:entity-boilerplate']; protected $interactive = TRUE }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private $stderr = class Symfony\Component\Console\Output\StreamOutput { private $stream = resource(96) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... } }; private $consoleSectionOutputs = []; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(89) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } } }) /Users/totten/src/civix/vendor/symfony/console/Application.php:273
    0.0314    2464456   6. CRM\CivixBundle\Command\AddEntityBoilerplateCommand->run($input = class Symfony\Component\Console\Input\ArgvInput { private $tokens = [0 => 'generate:entity-boilerplate']; private $parsed = []; protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $hasAnArrayArgument = FALSE; private $hasOptional = FALSE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = []; protected $arguments = ['command' => 'generate:entity-boilerplate']; protected $interactive = TRUE }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private $stderr = class Symfony\Component\Console\Output\StreamOutput { private $stream = resource(96) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... } }; private $consoleSectionOutputs = []; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(89) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } } }) /Users/totten/src/civix/vendor/symfony/console/Application.php:1009
    0.0314    2466288   7. CRM\CivixBundle\Command\AddEntityBoilerplateCommand->execute($input = class Symfony\Component\Console\Input\ArgvInput { private $tokens = [0 => 'generate:entity-boilerplate']; private $parsed = []; protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 1; private $hasAnArrayArgument = FALSE; private $hasOptional = FALSE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = []; protected $arguments = ['command' => 'generate:entity-boilerplate']; protected $interactive = TRUE }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private $stderr = class Symfony\Component\Console\Output\StreamOutput { private $stream = resource(96) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... } }; private $consoleSectionOutputs = []; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(89) of type (stream); private ${Symfony\Component\Console\Output\Output}verbosity = 32; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } } }) /Users/totten/src/civix/vendor/symfony/console/Command/Command.php:255
    0.2740   41521688   8. CRM\CivixBundle\Command\AddEntityBoilerplateCommand->CRM\CivixBundle\Command\{closure:/Users/totten/src/civix/src/CRM/CivixBundle/Command/AddEntityBoilerplateCommand.php:118-137}($generator = 'generateCreateSql', $fileName = 'auto_install.sql') /Users/totten/src/civix/src/CRM/CivixBundle/Command/AddEntityBoilerplateCommand.php:138
    0.2740   41521848   9. CRM_Core_CodeGen_Schema->generateCreateSql() /Users/totten/src/civix/src/CRM/CivixBundle/Command/AddEntityBoilerplateCommand.php:125
```

Still poking around at that...

**Comment by totten** [comment]
Should be fixed by #251. Published as v22.07.1.

**Comment by rbaugh** [comment]
@totten I pulled the latest release and it is working ok now. This was using PHP8.
